packages/integration-sdk-core/src/data/createIntegrationEntity.ts

```diff
@@ -103,7 +103,12 @@ export function createIntegrationEntity(
   validateRawData(generatedEntity);
 
   if (process.env.ENABLE_GRAPH_OBJECT_SCHEMA_VALIDATION) {
```

Comment:
I wonder if we should remove support for the env var.

Reply:
Meaning, during development we should always being logging the warnings but still not in production.